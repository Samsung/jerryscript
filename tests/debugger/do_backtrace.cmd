next
n
next
step
next
n
s
bt
c
