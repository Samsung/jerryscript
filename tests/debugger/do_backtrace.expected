Connecting to: localhost:5001
Stopped at tests/debugger/do_backtrace.js:15:1
(jerry-debugger) next
out: backtrace-test
Stopped at tests/debugger/do_backtrace.js:28:5
(jerry-debugger) n
Stopped at tests/debugger/do_backtrace.js:37:9
(jerry-debugger) next
Stopped at tests/debugger/do_backtrace.js:40:1
(jerry-debugger) step
Stopped at tests/debugger/do_backtrace.js:32:9 (in test() at line:30, col:1)
(jerry-debugger) next
out: function test
Stopped at tests/debugger/do_backtrace.js:33:9 (in test() at line:30, col:1)
(jerry-debugger) s
Stopped at tests/debugger/do_backtrace.js:23:3 (in foo() at line:21, col:1)
(jerry-debugger) bt
Frame 0: tests/debugger/do_backtrace.js:23:3 (in foo() at line:21, col:1)
Frame 1: tests/debugger/do_backtrace.js:33:12 (in test() at line:30, col:1)
Frame 2: tests/debugger/do_backtrace.js:40:5
(jerry-debugger) n
out: function foo
Stopped at tests/debugger/do_backtrace.js:24:7 (in foo() at line:21, col:1)
(jerry-debugger) n
Stopped at tests/debugger/do_backtrace.js:25:3 (in foo() at line:21, col:1)
(jerry-debugger) s
Stopped at tests/debugger/do_backtrace.js:18:9 (in f4() at line:17, col:1)
(jerry-debugger) backtrace
Frame 0: tests/debugger/do_backtrace.js:18:9 (in f4() at line:17, col:1)
Frame 1: tests/debugger/do_backtrace.js:25:5 (in foo() at line:21, col:1)
Frame 2: tests/debugger/do_backtrace.js:33:12 (in test() at line:30, col:1)
Frame 3: tests/debugger/do_backtrace.js:40:5
(jerry-debugger) c
out: function f4
